docker-compose.yml

```diff
@@ -1,37 +1,26 @@
---- 
-networks: 
-  firefly_iii_net: 
-    driver: bridge
-services: 
-  firefly_iii_app: 
+version: '3.3'
+
```

Comment:
I'm missing the network + driver config.

Reply:
I left it out on purpose, every service in the compose file will be defaulted to the `firefly_default` network which uses `bridge` as a default driver.